assets/src/blocks/SocialMedia/SocialMediaBlock.js

```diff
@@ -0,0 +1,50 @@
+import { SocialMediaEditor } from './SocialMediaEditorScript.js';
+import { socialMediaV1 } from './deprecated/socialMediaV1';
+import { frontendRendered } from '../frontendRendered.js';
+import { OEMBED_EMBED_TYPE, FACEBOOK_PAGE_TAB_TIMELINE } from './SocialMediaConstants.js';
+
+const { registerBlockType } = wp.blocks;
+const { __ } = wp.i18n;
+
+const BLOCK_NAME = 'planet4-blocks/social-media';
+
+export const registerSocialMediaBlock = () => registerBlockType(BLOCK_NAME, {
+  title: __('Social Media', 'planet4-blocks-backend'),
+  icon: 'share',
+  category: 'planet4-blocks',
+  attributes: {
+    title: {
+      type: 'string',
+      default: '',
+    },
+    description: {
+      type: 'string',
+      default: '',
+    },
+    embed_type: {
+      type: 'string',
+      default: OEMBED_EMBED_TYPE,
+    },
+    facebook_page_tab: {
+      type: 'string',
+      default: FACEBOOK_PAGE_TAB_TIMELINE,
+    },
+    social_media_url: {
+      type: 'string',
+      default: ''
+    },
+    alignment_class: {
+      type: 'string',
+      default: ''
+    },
+    embed_code: {
+      type: 'string',
+      default: '',
+    },
+  },
+  edit: SocialMediaEditor,
+  save: frontendRendered(BLOCK_NAME),
```

Comment:
Does this need to be front end renderer? :thinking:

Reply:
Hmmm yeah good point, I guess we could directly render `SocialMediaFrontend` 🤔

Reply:
I had a quick look at what these embeds contain, and I think we can directly save the html inside of the block as its content, instead of saving the same HTML in the attribute. Only thing to look out for is maybe the scripts some of these would add, which would then run earlier. For example the Twitter embed for me contained this after the `blockquote` element:
```
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>